tests/middleware/test_message_state.py

```diff
@@ -127,3 +129,16 @@ def do_work(x):
         args = state_backend.get_state(msg.message_id).args
         # if the max_size == 0, then should not storage nothing
         assert list(args) == expected
+
```

Comment:
these two tests should be one, however when I added the MagicMock and query the get_state(msg.message_id), it returned None due to the Mock

Reply:
don't really care about the fist one, second is sufficient.